packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -165,8 +165,8 @@ four optional attributes.
 new UserPool(this, 'myuserpool', {
   // ...
   requiredAttributes: {
-    fullname: true,
-    address: true,
+    fullname: new StandardAttribute(),
+    address: StandardAttribute.asMutable(),
```

Comment:
The new user experience can be made more slick and easy to use.

Something like this -

```ts
standardAttributes: {
  fullname: {
    required: true,
    mutable: false,
  },
  address: {
    required: false,
    mutable: true,
  },
}
```

We should mark `requiredAttributes` as deprecated and recommend the use of `standardAttributes`.

Reply:
:) that's a way better experience than my proposal, I'll do the changes over the weekend

Reply:
When you said we should mark requriedAttributes as deprecated you mean we should keep the property the same and simply set it as deprecated?
Can we set it as deprecated but point to the StandardAttribute interface instead? I know it breaks the interface but just wondering what exactly you meant there

Reply:
I meant to only mark it as deprecated and not change the interface